WordPress/src/test/java/org/wordpress/android/ui/jetpack/restore/RestoreViewModelTest.kt

```diff
@@ -1,24 +1,131 @@
 package org.wordpress.android.ui.jetpack.restore
 
+import android.os.Bundle
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.argThat
+import com.nhaarman.mockitokotlin2.whenever
 import kotlinx.coroutines.InternalCoroutinesApi
+import org.assertj.core.api.Assertions
 import org.junit.Before
 import org.junit.Test
 import org.mockito.Mock
+import org.mockito.Mockito
 import org.wordpress.android.BaseUnitTest
+import org.wordpress.android.ui.jetpack.restore.RestoreViewModel.RestoreWizardState
+import org.wordpress.android.ui.jetpack.restore.RestoreViewModel.RestoreWizardState.RestoreCanceled
+import org.wordpress.android.ui.jetpack.restore.RestoreViewModel.ToolbarState
+import org.wordpress.android.ui.jetpack.restore.RestoreViewModel.ToolbarState.DetailsToolbarState
 import org.wordpress.android.util.wizard.WizardManager
+import org.wordpress.android.viewmodel.SingleLiveEvent
+import java.util.Date
 
 @InternalCoroutinesApi
 class RestoreViewModelTest : BaseUnitTest() {
     @Mock lateinit var wizardManager: WizardManager<RestoreStep>
+    @Mock lateinit var restoreStep: RestoreStep
+    @Mock lateinit var savedInstanceState: Bundle
+    private val wizardManagerNavigatorLiveData = SingleLiveEvent<RestoreStep>()
 
     private lateinit var viewModel: RestoreViewModel
 
+    private val restoreState = RestoreState(
+            activityId = "activityId",
+            rewindId = "rewindId",
+            restoreId = 100L,
+            siteId = 200L,
+            url = null,
+            published = Date(1609690147756)
+    )
+
     @Before
     fun setUp() {
+        whenever(wizardManager.navigatorLiveData).thenReturn(wizardManagerNavigatorLiveData)
+        whenever(wizardManager.showNextStep()).then {
+            wizardManagerNavigatorLiveData.value = restoreStep
+            Unit
+        }
         viewModel = RestoreViewModel(wizardManager)
     }
 
     @Test
-    fun `sample test`() {
-    } // TODO:
+    fun `given view model, when started, then process moves to next step`() {
+        viewModel.start(null)
+
+        Mockito.verify(wizardManager).showNextStep()
+    }
+
+    @Test
+    fun `given in details step, when onBackPressed, then invokes wizard finished with cancel`() {
+        val wizardFinishedObserver = initObservers().wizardFinishedObserver
+
+        viewModel.start(null)
+        Mockito.clearInvocations(wizardManager)
```

Comment:
Idea (💡): Consider moving the `Mockito.` to imports. IMHO, it is not necessary to have it being replicated everywhere because Mockito related methods are distinguishable enough and used almost everywhere in every test suite. By moving that to imports section we also save line characters, which might help make a one-liner, where otherwise it will not fit.

PS: The very same apply for the `Assertions` import.

Reply:
in 65b23f3